app/helpers/application_helper.rb

```diff
@@ -79,4 +79,39 @@ def flash_message(type, text, now = nil)
       flash[type] << text
     end
   end
+
+  def rental_status_css_class(rental)
+    if rental.rental_status == 'checked_out' && rental.end_time < Time.current
+      return 'danger'
+    elsif rental.rental_status == 'checked_out'
+      return 'info'
+    elsif rental.rental_status == 'canceled'
+      return ''
+    elsif rental.rental_status != 'checked_out' && rental.start_time < Time.current
+      return 'warning'
+    elsif rental.rental_status == 'reserved' && rental.start_time.to_date == Time.zone.today
+      return 'success'
+    elsif rental.rental_status == 'reserved' && rental.start_time > Time.current
+      return 'active'
+    else
+      return ''
+    end
+  end
+
+  def rental_status_english(status)
```

Comment:
Keep in mind that the return value of a method is the value output by the last expression. This is to say, this method could just be:

```ruby
def rental_status_english(status)
  case status
  when 'active' then 'Reserved future'
  # ...
end
```

although, truth be told, this just seems like a mapping from string values to string values, and could be accomplished best perhaps with a Hash.